client/aws/errors.go

```diff
@@ -11,8 +11,23 @@ const (
 	ELBConfigurationMismatch = "already exists and it is configured with different parameters"
 )
 
-var (
-	malformedAmazonAccountIDError   = errgo.New("amazon account ID can only contain numbers")
-	wrongAmazonAccountIDLengthError = errgo.New("amazon account ID has the wrong size")
-	emptyAmazonAccountIDError       = errgo.New("amazon account ID cannot be empty")
-)
+var malformedAmazonAccountIDError = errgo.New("malformed amazon account ID, it should only contain numbers")
```

Comment:
Can you show me how the name `malformed amazon account id` doesn't match with the message `malformed amazon account ID...`? You mean that I should get rid of the last part of message? If yes, why in your opinion a message cannot add some additional explanation about an error?